python/hail/plot/plots.py

```diff
@@ -133,3 +231,98 @@ def qq(pvals):
     bound = max(max(exp), max(obs)) * 1.1
     p.line([0, bound], [0, bound], color='red')
     return p
+
+
+@typecheck(pvals=expr_float64, locus=nullable(expr_locus()), title=nullable(str),
+           size=int, hover_fields=nullable(dictof(str, sequenceof(anytype))))
+def manhattan(pvals, locus=None, title=None, size=4, hover_fields=None):
+    """Create a Manhattan plot. (https://en.wikipedia.org/wiki/Manhattan_plot)
+
+    Parameters
+    ----------
+    pvals : :class:`.Float64Expression`
+        P-values to be plotted.
+    locus : :class:`.LocusExpression`
+        Locus values to be plotted.
+    title : str
+        Title of the plot.
+    size : int
+        Size of markers in screen space units.
+    hover_fields: Dict[str, List[Any]]
+        Dictionary of field names and values to be shown in the HoverTool of the plot.
+
+    Returns
+    -------
+    :class:`bokeh.plotting.figure.Figure`
+    """
+    def get_contig_index(x, starts):
+        left = 0
+        right = len(starts) - 1
+        while left <= right:
+            mid = int((left + right) / 2)
```

Comment:
Python has a floor division operator `//` which would do the same thing. But this is fine.

Reply:
though if you're changing other stuff might as well.